cli/cmd/component_args_test.go

```diff
@@ -75,6 +75,45 @@ func TestComponentArgs(t *testing.T) {
 				"--debug",
 			},
 		},
+		{
+			[]string{
+				"iac", "negative-values", "--time", "-24h",
+			},
+			[]string{
+				"iac", "negative-values", "--time", "-24h",
+			},
+			[]string{},
+		},
+		{
+			// We do not have access to the component flag configuration and therefore cannot
+			// support the shorthand expansion of `-xyz` -> `-x -y -z`.  We have to treat it as
+			// `x=yz` and pass through to the component.
```

Comment:
Dropping support of shorthand expansion is fine. It just limits our "expert" users to chain flags. If we get
complains, we have a way forward, and maybe in the future we switch to something similar to https://github.com/lacework/go-sdk/pull/1508